src/components/reusuables/sections/YohdevSection.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import YohDevButton from "../../YohDevButton"
+import SectionHeader from "../typography/SectionHeader"
+
+const YohdevSection = (props) => {
+  return (
+    <section className={`yohdev-section-container ${props.style}`}>
+      <Container>
+        {props.sectionHeading == "true" ? (
+          <div className={`text-container mb-5 ${props.color}`}>
+            <SectionHeader className="test" heading={props.heading} />
+            <p className={`section-text`}>{props.text}</p>
+          </div>
+        ) : (
+          ""
+        )}
+
+        {props.children}
+
+        {props.button == "true" ? (
```

Comment:
Can we set this logic at the top like we discussed in our call? Change true to boolean instead of string.

cont number = number === true ? number : false;
{number && (

)
}

Reply:
updated logic